jobs/elasticsearch/monit

```diff
@@ -1,8 +1,19 @@
+<% if p("elasticsearch.monitor.type") == "process") %>
 check process elasticsearch
   with pidfile /var/vcap/sys/run/elasticsearch/elasticsearch.pid
   start program "/var/vcap/jobs/elasticsearch/bin/monit_debugger elasticsearch_ctl '/var/vcap/jobs/elasticsearch/bin/elasticsearch_ctl start'" with timeout 120 seconds
   stop program "/var/vcap/jobs/elasticsearch/bin/monit_debugger elasticsearch_ctl '/var/vcap/jobs/elasticsearch/bin/elasticsearch_ctl stop'"
   group vcap
+<% else if p("elasticsearch.monitor.type") == "host" %>
+check host elasticsearch with address 127.0.0.1
+  start program "/var/vcap/jobs/elasticsearch/bin/monit_debugger elasticsearch_ctl '/var/vcap/jobs/elasticsearch/bin/elasticsearch_ctl start'" with timeout 120 seconds
+  stop program "/var/vcap/jobs/elasticsearch/bin/monit_debugger elasticsearch_ctl '/var/vcap/jobs/elasticsearch/bin/elasticsearch_ctl stop'"
+  if failed url http://127.0.0.1:9200/_cluster/health
```

Comment:
That's a good point, the master will block until it can form a quorum.  WDYT about only enabling the content check if  `p("elasticsearch.node.allow_master") == false` ?  

That will still cause clusters to go red for people running running combo data and master nodes, but if they are doing that then they really don't care about running ES right anyway :trollface: